### DISCUSSION TITLE
How to compress PNG image? Maximum compression lossless quality

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [dlemstra/Magick.NET#980] [Help]
Hello. I would like to compress PNG images what is the command?

I want to make it with lossless quality

how much time or CPU it takes doesn't matter

Previously I have used PNGOUTWin  but I think it is now outdated

**Comment by dlemstra** [comment]
The `PNG` encoder is always lossless. You could compress the image with the `ImageOptimizer` class. But  you might also want to consider other tools. A tool I heared about is `optipng`.
